glib/src/types.rs

```diff
@@ -297,6 +297,33 @@ macro_rules! builtin {
     };
 }
 
+// rustdoc-stripper-ignore-next
+/// A GLib [pointer](ffi::gpointer)
+pub type Pointer = *mut Pointee;
```

Comment:
```suggestion
pub type Pointer = ffi::pointer;
```

Why not this?

Reply:
I mostly just wanted the docs to be self-descriptive here with a link to `Pointee`, clearly demonstrating the relationship between the two types. It does happen to be equivalent to `ffi::gpointer` so it could go either way, but I was kind of going for an almost "opaque" wrapper despite it not actually being a newtype - which is to say I'm somewhat discouraging its use directly in FFI code as a replacement for `ffi::gpointer`.
If you wanted to be insane, this could actually be defined as `type Pointer = Option<NonNull<Pointee>>` instead :stuck_out_tongue:

Reply:
I'd prefer to go via `ffi::gpointer` then as that's what is used elsewhere for this :)